app/src/main/java/org/mozilla/focus/settings/privacy/TrackingProtectionPanel.kt

```diff
@@ -10,6 +10,7 @@ import androidx.appcompat.content.res.AppCompatResources
 import androidx.core.view.isVisible
 import com.google.android.material.bottomsheet.BottomSheetBehavior
 import com.google.android.material.bottomsheet.BottomSheetDialog
+import kotlinx.android.synthetic.main.dialog_tracking_protection_sheet.*
```

Comment:
nit: do we need this import?

Reply:
It added itself while I was typing the view id. I think it knows I plan to remove synthetics. 🤖 😄